models/place.js

```diff
@@ -0,0 +1,16 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+// mongoose.connect('mongodb://localhost/ibi-ironhack',  {useMongoClient: true});
+// mongoose.Promise = require('bluebird');
+
+const placeSchema = new Schema({
+    name: {type: String},
+    location: {
+                lat: {type: String},
+                lng: {type: String},
+            },
+    type: {type: String}
```

Comment:
First of all, type is an awful name for the kind of store. That said, if there are a list of possible values, you can use enum:

```javascript
bussines: {
  type: String,
  enum : ['coffe','bookstore'],
}
```